src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -148,9 +148,24 @@ Configuring `edgeMiddleware: true` will deploy your middleware as an Edge Functi
 
 ### Netlify Image CDN support
 
-This adapter uses the [Netlify Image CDN](https://docs.netlify.com/image-cdn/) to transform images on-the-fly without impacting build times.
+This adapter by default uses the [Netlify Image CDN](https://docs.netlify.com/image-cdn/) to transform images on-the-fly without impacting build times.
```

Comment:
The URL https://docs.netlify.com/image-cdn/ 404’s - it should be https://docs.netlify.com/image-cdn/overview/

Was going to open a PR for it but might as well catch it here if you can @alexanderniebuhr